students/carlos_novoa/lesson02/fizzbuzz.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+
+"""
+Lesson2, FizzBuz
+"""
+
+
+def fizzbuzz():
+    for i in range(1, 101):
+        if (i % 5 == 0 and i % 3 == 0):
```

Comment:
Forgot to mention in my review earlier that `if` statements don't require parentheses in Python so these can all be rewritten as:

`if i % 5 == 0 and i % 3 == 0:`

Reply:
Much more clean and Pythonic, thanks